Fabric.Terminology.API/Scripts/Terminology-Install-Utilities.psm1

```diff
@@ -52,33 +52,110 @@ function Get-ConfigValue {
     }
 }
 
-function Invoke-SqlCommand{
+function Get-ServiceFromDiscovery {
+    param(
+        [String] $DiscoveryUrl = $(throw "Please specify the Discovery Service Uri"),
```

Comment:
I'm ok with this, but couldn't you just use `[ValidateNotNullorEmpty()]`

Reply:
I can (and probably should).  I saw that used in Imran's email but I had already been validating this way and forgot.  `[ValidateNotNullorEmpty()]` would be better.  I'll change it.

Reply:
I agree with validating parameters, but there is a cleaner way